builder/pipeline.cc

```diff
@@ -680,7 +680,10 @@ class pipeline_builder {
       // And make the actual loop.
       expr loop_step = sanitizer_.mutate(loop.step);
       interval_expr loop_bounds = get_loop_bounds(base_f, loop);
-      body = loop::make(loop.sym(), loop.max_workers, loop_bounds, loop_step, body);
+      // Make sure that a loop variable is unique.
+      var loop_var = ctx.insert_unique(ctx.name(loop.sym()));
```

Comment:
It'd be nice to be able to give this a more meaningful name, e.g. `func_name.loop_var` (like Halide does). However, I'm not sure we actually have a good thing to use here, so only do this if there's something easy we can do here.

Reply:
I'm using attrs.name now, I'm not sure I liked it, because it can be an arbitrary string (with spaces etc), but take a look.